layout/templates/keynote.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="keynote" id="{{.}}">
```

Comment:
I would change this to 
```
<div class="keynote" id="{{id}}">
  <div class="keynoteContent">{{content}}</div>
  <div class="dimmer"></div>
</div>
```
and then make the corresponding change to siteText.yaml (see other comment)